denizen_scripts/global/server/shapes.dsc

```diff
@@ -287,4 +287,4 @@ play_animation:
   - define points <[points]>
   - repeat <[points].size>:
     - playeffect <[particle]> <[points].get[<[value]>]> offset:0 visibility:300 quantity:2
-    - wait 1t
+    - wait 1t
```

Comment:
newline at the end of file prevents difference formatting like this looking strange
```suggestion
    - wait 1t

```